test/app-luatest/gh_7808_rel_paths_in_diag_and_debug_info_test.lua

```diff
@@ -0,0 +1,25 @@
+local bit = require('bit')
+local misc = require('test.luatest_helpers.misc')
+local netbox = require('net.box')
+local t = require('luatest')
+
+local g = t.group()
+
+g.test_rel_paths_in_diag = function(_)
+    local _, err = pcall(function() netbox.self:call('') end)
+    t.assert_not(err.trace[1].file:startswith('/'))
+end
+
+g.test_rel_paths_in_debug_info = function(_)
+    misc.skip_if_macOS()
```

Comment:
There's also FreeBSD, where this test will probably not work, either. I think we should do `skip_if_not_linux` here.

Reply:
Removed this helper in favor of testing for `readline` availability.